src/Microsoft.AspNetCore.Mvc.Core/Internal/ControllerActionInvoker.cs

```diff
@@ -45,8 +43,7 @@ public class ControllerActionInvoker : ResourceInvoker, IActionInvoker
             ILogger logger,
             DiagnosticSource diagnosticSource,
             ControllerContext controllerContext,
-            IFilterMetadata[] filters,
-            ObjectMethodExecutor objectMethodExecutor)
+            IFilterMetadata[] filters)
```

Comment:
Actually, it looks like with these changes, there's no possible way for external code to use their own instantiated `ControllerActionInvoker` instances, because they have no way to assign `Executor`, and so they can't call `InvokeActionMethodAsync` without getting an exception.

As such, I agree it definitely makes sense at least to mark the constructor (if not the whole type) as internal, so people don't get confused about the correct way to use the type.